guides/common/modules/proc_setting-up-pull-client.adoc

```diff
@@ -0,0 +1,34 @@
+[id="setting-up-pull-client_{context}"]
+= Setting Up the Remote Execution Pull Client
+
+For {SmartProxies} in `pull-mqtt` mode, hosts can subscribe to remote jobs using the remote execution pull client.
+No SSH connection is required.
+
+Use this procedure to configure a host to use the pull client:
```

Comment:
This line does not really match the title. Maybe reword the title and drop this line.